### PR TITLE
shapes

### PR DESCRIPTION
**Pull request by gintama91**

### CODE REVIEW

**Comment by gintama91** [comment]
image

**Comment by gintama91** [comment]
ah yeah i used star branch as base hm #196

**Comment by gintama91** [comment]
hm aftr #196 got merged i will look into linter my bad my 4 spaces intendation by default i forgot to change that!
i will try to improvise example if needed maybe with more diff shapes!

File: lib/scarpe/wv/shape_helper.rb [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module ShapeHelper
```
Comment (gintama91) [inline lib/scarpe/wv/shape_helper.rb:3]: i wanted move_to , line_to commands in nrml file and shape_path commmand in wv directory so i made a seperate module for these methods

Comment (noahgibbs) [reply lib/scarpe/wv/shape_helper.rb:3]: Sounds good. We'll probably want more of that kind of thing going forward. I've been looking at shoes-core in shoes4 and it does quite a lot of it...

File: lib/scarpe/wv/shape_helper.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+      raise ArgumentError, "Invalid coordinates: x=#{x}, y=#{y}" unless valid_coordinate?(x) && valid_coordinate?(y)
+    end
+  
+    def valid_coordinate?(coordinate)
+      coordinate.is_a?(Numeric)
+    end
+  
+    def check_path_commands(path_commands_str)
+      if path_commands_str.empty?
+        puts "ah oh Looks like something is wrong. No path commands found."
```
Comment (noahgibbs) [inline lib/scarpe/wv/shape_helper.rb:36]: Should this be an error or return a boolean or something? Just printing to console is easy to get lost.

Comment (gintama91) [reply lib/scarpe/wv/shape_helper.rb:36]: hm i added tht just incase , not an error as of now but might be

Comment (gintama91) [reply lib/scarpe/wv/shape_helper.rb]: ok i removed this method

File: lib/scarpe/wv/shape.rb [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    private
+
+    def style
+      {
+        width: "400",
+        height: "900",
+      }
+    end
+
+    def stroke_color
```
Comment (noahgibbs) [inline lib/scarpe/wv/shape.rb:33]: It feels like this should be configurable. But I won't worry more about it for now. If I care that much I can add that feature later :-)

**Pull request merged**